bin/antigen.zsh

```diff
@@ -121,9 +121,8 @@ antigen () {
   # used to create the given directory path.
 
   if [[ "$1" == ".zprezto" ]]; then
-    # Prezto's (in `.zprezto`), is assumed to be from `sorin-ionescu`'s
-    # remote.
-    echo https://github.com/sorin-ionescu/prezto.git
+    # Prezto's (in `.zprezto`), is assumed to be from the community fork.
+    echo https://github.com/zsh-users/prezto.git
```

Comment:
We should move this to a configuration variable, defaulting to `https://github.com/zsh-users/prezto.git`. So they can easily switch between prezto forks, if needed.